domain/blockdag/ghostdag_test.go

```diff
@@ -188,41 +66,42 @@ func TestGHOSTDAG(t *testing.T) {
 			genesisNode := dag.genesis
 			blockByIDMap := make(map[string]*blockNode)
 			idByBlockMap := make(map[*blockNode]string)
-			blockByIDMap["A"] = genesisNode
-			idByBlockMap[genesisNode] = "A"
+			blockByIDMap[test.GenesisID] = genesisNode
+			idByBlockMap[genesisNode] = test.GenesisID
 
-			for _, blockData := range test.dagData {
+			for _, blockData := range test.Blocks {
 				parents := blockSet{}
-				for _, parentID := range blockData.parents {
+				for _, parentID := range blockData.Parents {
 					parent := blockByIDMap[parentID]
 					parents.add(parent)
 				}
 
 				block, err := PrepareBlockForTest(dag, parents.hashes(), nil)
 				if err != nil {
-					t.Fatalf("TestGHOSTDAG: block %v got unexpected error from PrepareBlockForTest: %v", blockData.id, err)
+					t.Fatalf("TestGHOSTDAG: block %v got unexpected error from PrepareBlockForTest: %v", blockData.ID,
```

Comment:
Please use %s for strings rather than %v, here and everywhere.

Reply:
Done